assets/hub-search-bar.js

```diff
@@ -20,7 +21,9 @@ $("#hub-search-icon").on("click", function() {
 
 function hideHubSearch(searchIcon) {
   $(searchIcon).hide();
+  
   $("#hub-search-icon, #dropdown-filter-tags").fadeIn("slow");
+  $("#hub-icons").fadeIn("slow");
```

Comment:
I think you need to add this ID to the icons on the compact/regular hub index pages. Right now the icons aren't fading in on those pages.